scripts/depcheck/missing.ts

```diff
@@ -67,7 +67,7 @@ const IGNORE_MATCHES_IN_PACKAGE = {
     'identity-obj-proxy',
     '@angular-devkit/schematics',
   ],
-  js: [],
```

Comment:
We should put an optional peer dep in @nx/js/package.json and then this shouldn't be needed.